src/styles/theme.css

```
@@ -0,0 +1,29 @@
+  background-color: #ffffff;
+  color: #111111;
+}
+
+body.dark {
+  background-color: #121212;
+  color: #f5f5f5;
+}
+
+button {
+  cursor: pointer;
+  border: 1px solid currentColor;
```

Stylelint (17.7.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)